requirements-dev.txt

```diff
@@ -18,4 +18,5 @@ cmake
 custatevec-cu12
 cutensornet-cu12
 pylint==2.7.4
+scipy-openblas32
```

Comment:
I suggest adding the same dependency to `requirements.txt` otherwise will fail the compilation.

Reply:
Do you mean `pip editable install`?

Reply:
Nop. If we follow the installation [instructions](https://github.com/PennyLaneAI/pennylane-lightning/blob/158d39309657b8aa9d512eea3b2ebb841f854415/README.rst?plain=1#L126-L130) then the process shows the following error:
``` bash 
  CMake Error at cmake/FindSCIPY_OPENBLAS32.cmake:22 (message):


    Unable to find libscipy_openblas.so.  Please set a SCIPY_OPENBLAS32 env
    variable to provide the path to scipy_openblas32.
  Call Stack (most recent call first):
    CMakeLists.txt:109 (find_path_to_openblas)

```

Reply:
Thanks @LuisAlfredoNu. the instruction you shared is editable installation. We have to update the instruction. CC: @maliasadi